modules/cluster-logging-vector-tech-preview.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-vector-tech-preview_{context}"]
+:FeatureName: Vector
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-vector"]
+= About Vector
+[role="_abstract"]
+Vector is an observability data pipeline currently offered as an alternative to the default collector for the {logging}.
```

Comment:
Vector is a log collector  offered as a tech-preview alternative to the current default ...